crates/ott-balancer/src/lib.rs

```diff
@@ -57,6 +57,19 @@ pub async fn run() -> anyhow::Result<()> {
 
     let config = BalancerConfig::get();
 
+    if config.selection_strategy.is_some() {
+        info!("Using selection strategy: {:?}", config.selection_strategy);
+        BalancerContext {
+            monolith_selection: config
+                .selection_strategy
+                .expect("selection strategy not set"),
+            ..BalancerContext::default()
+        }
+    } else {
+        info!("Using default selection strategy");
+        BalancerContext::default()
+    };
```

Comment:
This `BalancerContext` is not actually being used, and this needs to happen after the `tracing_subscriber::registry().init()` otherwise these log lines will never appear, so it needs to go after the "Loaded config" log line

Reply:
Just to clarify before I make this change, the config isn't fully loaded until here `info!("Loaded config: {:?}", config);` because it is being manipulated in some way. `BalancerConfig::load` just allows it to be manipulated from the file path.

Reply:
No the config is loaded at `BalancerConfig::load()`. The `info!` log macros don't work until tracing subscriber is initialized.